src/iosource/pcap/Source.h

```diff
@@ -44,6 +45,8 @@ class PcapSource : public PktSrc
 
 	pcap_t* pd;
 	struct pcap_stat prev_pstat = {0};
+
+	std::vector<char> iobuf;
```

Comment:
This should have a comment documenting what this is used for under what circumstances.